summarize.go

```diff
@@ -168,12 +171,12 @@ func printVal(cfg Config, value reflect.Value, indent string) string {
 		}
 
 	case v.CanInterface():
-		v := v.Interface()
-		switch v.(type) {
+		iFace := v.Interface()
```

Comment:
This change isn't necessary, but shadowing `v` was confusing when trying to read the debugger output when i was working on this.

Reply:
I was confused with that last time as well, after a small conversation with keith I decided to keep the current pattern the repo follows (keep shadowing, since this is everywhere in this codebase). I realized I'm just not used to seeing shadowing in the debugger on the regular, but now know what to look for. (I'm a +0 on the rename, with a preference to stay consistent in the repo, but consider this a non-blocking comment)

Reply:
I've reverted the rename. At some point I might make a PR that completely removes shadowing from this repo, since I think it's confusing and doesn't add much, but we can table that discussion for a different day.